components/inputs/input-time.js

```diff
@@ -306,8 +326,9 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 					@d2l-dropdown-close="${this._handleDropdownClose}"
 					@d2l-dropdown-open="${this._handleDropdownOpen}"
 					no-padding-footer
-					max-height="${ifDefined(this.maxHeight)}"
-					min-width="195">
+					max-height="${this._dataMobile ? null : ifDefined(this.maxHeight)}"
```

Comment:
Again, should `d2l-dropdown-menu` maybe just be ignoring its `max-height` if `mobile-tray` is set and it's put into mobile mode?